### PR TITLE
cleaner version of the repo to be checked

### PR DESCRIPTION
**Pull request by SimonCalo**

### CODE REVIEW

File: analysis/Scripts/FitScript.py [hunk 1]
```diff
@@ -4,8 +4,10 @@
 from ROOT import TChain, TFile
 from Universal_Mass_Fit_function import Shape_fit
 
-user = "Simon"
-shape = "Bukin" # select the shape that you want to fit your data with
+#user = "Simon"
+user = input("please indicate the user (Simon, Chris or Nikhef): ")
+#shape = "Bukin" # select the shape that you want to fit your data with
+shape = input("please indicate the PDF (GaussCB or Bukin): ")
```
Comment (jackfrost373) [inline analysis/Scripts/FitScript.py]: Is this really beneficial? If you want to run it 10 times to fix some bugs, you have to manually enter your name 10 times...

File: analysis/Scripts/Imports.py [hunk 2]
```diff
@@ -3,7 +3,8 @@
 
 ROOT.gStyle.SetOptStat(0)
 
-user = "Nikhef"
+#user = "Nikhef"
+user = input("Please indicate the user (Simon, Chris or Nikhef): ")
```
Comment (jackfrost373) [inline analysis/Scripts/Imports.py]: Same here. Not sure it's such a great thing.

**Comment by SimonCalo** [comment]
@cpawley Please take a look at it before our skype of next week (unless I can manage to do what we set out to do before then). I am sure there are some minor mistakes here and there. I hope in general, it looks alright. Take a closer look especially at the tuples preparation scripts since they are brand new. Let me know how you would like me to change things. Cheers

File: analysis/Scripts/tuples_prep_scripts/SetBranch.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import ROOT, os
+from ROOT import TChain, TFile
+
+#### This function takes a ROOT file as an input, keeps the variables in useful_vars in the tree and throws the other ones away. The pruned tree is then returned. ###
+
+def setBranch_funct (root_file):
+
+    useful_vars = ["lcplus_MM", "lcplus_P", "lcplus_PT", "lcplus_ETA", "lcplus_RAPIDITY", "lcplus_TIP", "lcplus_IPCHI2_OWNPV" "lcplus_L0_Global_TOS", "lcplus_Hlt1Global_TOS", "lcplus_Hlt2Global_TOS", "lcplus_Hlt1Phys_TOS", "lcplus_Hlt2Phys_TOS", "lcplus_Hlt1TrackAllL0Decision_TOS", "lcplus_L0HadronDecision_TOS", "pplus_P", "pplus_PT", "pplus_RAPIDITY", "pplus_ETA", "piplus_P", "piplus_PT", "piplus_RAPIDITY", "piplus_ETA", "kminus_P", "kminus_PT", "kminus_RAPIDITY", "kminus_ETA"] # list of variables kept in the tree
```
Comment (cpawley) [inline analysis/Scripts/tuples_prep_scripts/SetBranch.py]: @cpawley to check

**Pull request merged**